x/tokenomics/keeper/settle_pending_claims.go

```diff
@@ -48,285 +62,50 @@ func (k Keeper) SettlePendingClaims(ctx cosmostypes.Context) (
 	for ; expiringClaimsIterator.Valid(); expiringClaimsIterator.Next() {
 		claim, iterErr := expiringClaimsIterator.Value()
 		if iterErr != nil {
-			return settledResults, expiredResults, iterErr
-		}
-		numExpiringClaims++
-
-		if err = settlementContext.ClaimCacheWarmUp(ctx, &claim); err != nil {
-			return settledResults, expiredResults, err
-		}
-
-		var (
-			// These values are used for cores business logic; minting, slashing, etc.
-			proofRequirement prooftypes.ProofRequirementReason
-			claimeduPOKT     cosmostypes.Coin
-
-			// These values are used for basic validation, telemetry and observability.
-			numClaimRelays,
-			numClaimComputeUnits,
-			numEstimatedComputeUnits uint64
-		)
-
-		sessionId := claim.GetSessionHeader().GetSessionId()
-
-		// DEV_NOTE: Not every Relay (Request, Response) pair in the session is inserted into the tree.
-		// See the godoc for GetNumRelays for more delays.
-		numClaimRelays, err = claim.GetNumRelays()
-		if err != nil {
-			return settledResults, expiredResults, err
-		}
-
-		// Ensure that the number of relays claimed is greater than 0.
-		// CreateClaim message handler validation already ensures that the number of relays
-		// is greater than 0, but this is a sanity check to ensure that no invalid claims
-		// are processed in the settlement phase.
-		if numClaimRelays == 0 {
```

Comment:
Why did we remove this?

Reply:
https://github.com/pokt-network/poktroll/pull/1441#discussion_r2131269305